server/index.js

```diff
@@ -316,6 +319,7 @@ const loop = () => {
         x: player.x,
         y: player.y,
         hp: player.hp,
+        energy: player.energy,
```

Comment:
I was thinking of sending single socket to the player similarly to respawn. Not to send energy information to all of the players and show only to one but send information only to one.
https://github.com/neu5-przenz-games/the-game/blob/905605fc3b393d206477aab5c08bd47aba5d0995/client/events/sockets.js#L148